### PR TITLE
Add recommendation notes for equivalence properties

### PR DESCRIPTION
**Pull request by csuwildcat**
***
<a href="https://pr-preview.s3.amazonaws.com/csuwildcat/did-core/pull/542.html" title="Last updated on Jan 24, 2021, 4:28 PM UTC (f1cd363)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/542/c8bc6b1...csuwildcat:f1cd363.html" title="Last updated on Jan 24, 2021, 4:28 PM UTC (f1cd363)">Diff</a>

### CODE REVIEW

File: index.html
Comment (msporny) [inline index.html:4031]:
```suggestion
arise related to user experience. Implementers are strongly advised to observe the 
```

Make the 'You' party more explicit.

File: index.html
Comment (msporny) [inline index.html:3964]:
```suggestion
arise. Implementers are strongly advised to observe the 
```

The negative issues aren't just related to user experience, they're also related to security. Make the 'You' party more explicit.

File: index.html
Comment (msporny) [inline index.html:3963]:
```suggestion
logically equivalent, there might be negative or unexpected issues that 
```

**Review by msporny — APPROVED** [review]
Minor nits otherwise LGTM.

File: index.html
Comment (msporny) [inline index.html:3962]: Also, to be clear -- Editor hat off -- equivalentId places a possibly unacceptable burden on RPs. I doubt Digital Bazaar will support any DID Method that uses the feature as it makes the lookup tables for account information very complex and potentially error prone. 

I expect other RP implementers to get the implementation wrong, which will result in security failures that are not the failure of the DID Method, but will most likely be blamed on the DID Method. I don't expect that canonicalId has the same issue.

**Comment by msporny** [comment]
Waiting for @csuwildcat to respond/approve changes before merging.

**Review by peacekeeper — APPROVED** [review]
Like @msporny I'm also a bit concerned that RPs may not implement these features correctly (speaking from experience with [XRI Resolution](http://docs.oasis-open.org/xri/2.0/specs/xri-resolution-V2.0.html) long ago, which also had multiple equivalence properties).

But considering that we already accepted these features, I think this particular PR is helpful to explain how to handle them correctly.

**Comment by peacekeeper** [comment]
BTW after merging this one, I propose that we create a separate PR to change all occurrences of "resolving party" to "requesting party". I created issue https://github.com/w3c/did-core/issues/563 for this.

**Review by OR13 — CHANGES_REQUESTED** [review]
Would like to see the editorial changes proposed by @msporny .

File: index.html
Comment (brentzundel) [inline index.html:4030]:
```suggestion
values as secondary aliases, there might be negative or unexpected issues that 
```

**Review by brentzundel — CHANGES_REQUESTED** [review]
one suggested change from me, and I'd also like to see Manu's suggestions accepted.

File: index.html
Comment (msporny) [reply index.html]: @brentzundel I think your change got applied when I merged some other stuff... if it didn't, we'll fix it.

**Comment by msporny** [comment]
> Waiting for @csuwildcat to respond/approve changes before merging.

No response from @csuwildcat in 5 days -- waiting on this PR is jamming up a bunch of other PRs. I'm applying changes (as others seemed to want them as well) and merging.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested (and merged by Editor due to non-response from original PR contributor), no objections, merging.

File: index.html
Comment (msporny) [reply index.html]: Just checked, it did.

**Comment by csuwildcat** [comment]
I'll be sure to open a new PR with any necessary changes.

**Comment by csuwildcat** [comment]
> Like @msporny I'm also a bit concerned that RPs may not implement these features correctly (speaking from experience with [XRI Resolution](http://docs.oasis-open.org/xri/2.0/specs/xri-resolution-V2.0.html) long ago, which also had multiple equivalence properties).
> 
> But considering that we already accepted these features, I think this particular PR is helpful to explain how to handle them correctly.

The RP has a primary DB column for users, as they do now, and they have a column for user IDs. The difference is allowing the column to have multiple values and a slightly different DB query. If that's too much for a DID consumer, heaven help these people when they are juggling all the crypto and data format craziness that makes this look like a cake walk.

**Pull request merged**
